core/src/it/scala/org/ensime/fixture/SearchServiceFixture.scala

```diff
@@ -17,8 +18,8 @@ trait IsolatedSearchServiceFixture extends IsolatedSourceResolverFixture {
       testCode(config, searchService)
     } finally {
       Await.ready(searchService.shutdown(), Duration.Inf)
-      actorSystem.shutdown()
-      actorSystem.awaitTermination(10.seconds)
+      actorSystem.close()
```

Comment:
I was thinking you could make the BackCompat only for 2.10 and use the akka 2.4 names... that way you don't need to write an impl for 2.11+